packages/front-end/components/Experiment/DateResults.tsx

```diff
@@ -95,7 +97,9 @@ const DateResults: FC<{
 
     const sortedResults = [...results];
     sortedResults.sort((a, b) => {
-      return getValidDate(a.name).getTime() - getValidDate(b.name).getTime();
+      return (
+        getValidDateUTC(a.name).getTime() - getValidDateUTC(b.name).getTime()
```

Comment:
It's probably actually not needed in these sort function (here and above in L70) since it'll just be doing extra offsetting to both dates. But it doesn't matter.

Reply:
cool i can remove it from here and l70